nonlib/OSC/Endpoint.C

```diff
@@ -1092,7 +1106,7 @@ namespace OSC
     }
 
     Signal *
-    Endpoint::add_signal ( const char *path, Signal::Direction dir, float min, float max, float default_value, signal_handler handler, void *user_data )
+    Endpoint::add_signal ( const char *path, Signal::Direction dir, float min, float max, float default_value, signal_handler handler, signal_feedback_handler feedback_handler, void *user_data )
```

Comment:
I had to change the add_signal's method signature to implement the update mechanism, hence the change in `midi-mapper.C`. Other non-apps will not be able to build againt this version of `nonlib/OSC` without similar changes.